src/js/modules/RoutingApp.js

```diff
@@ -40,6 +40,8 @@ class RoutingApp extends Component {
   }
 
   componentDidMount() {
+    const loadingIcon = document.querySelector(".loading");

```

Comment:
Oh my god what terrible code. What idiot wrote this?

Reply:
you did